csharp/extractor/Semmle.Extraction/TrapWriter.cs

```diff
@@ -190,7 +190,7 @@ public void Dispose()
                     FileUtils.TryDelete(tmpFile);
                 }
             }
-            catch (Exception ex)
+            catch (Exception ex)  // lgtm[cs/catch-of-all-exceptions]
```

Comment:
Perhaps we should disable this query in the `.lgtm.yml` file instead?

Reply:
I think leave this, because it would be in principle a good idea to receive new alerts for `cs/catch-of-all-exceptions` if we introduce new ones.